app/src/main/java/umc/everyones/lck/presentation/mypage/MyPageProfileFragment.kt

```diff
@@ -0,0 +1,141 @@
+            "Gold" to R.drawable.shape_circle_gold,
+            "Master" to R.drawable.shape_circle_master,
+            "Challenger" to R.drawable.shape_circle_challenger
+        )
+
+        val tierStyles = mapOf(
+            "Bronze" to R.style.TextAppearance_Bronze,
+            "Silver" to R.style.TextAppearance_Silver,
+            "Gold" to R.style.TextAppearance_Gold,
+            "Master" to R.style.TextAppearance_Master,
+            "Challenger" to R.style.TextAppearance_Challenger
+        )
+
+        // 기본 상태로 초기화 필요 없음
+
+        // 사용자 티어에 해당하는 요소만 업데이트
+        when (tier) {
+            "Bronze" -> {
+                binding.viewMypageProfileCircleBronze.setBackgroundResource(tierBackgrounds["Bronze"] ?: R.drawable.shape_circle_bronze)
+                binding.tvMypageProfileBronzeText.setTextAppearance(requireContext(), tierStyles["Bronze"] ?: R.style.TextAppearance_LCK_Light_Detail_10sp)
+                binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles["Bronze"] ?: R.style.TextAppearance_Bronze)
+            }
+            "Silver" -> {
+                binding.viewMypageProfileCircleSilver.setBackgroundResource(tierBackgrounds["Silver"] ?: R.drawable.shape_circle_silver)
+                binding.tvMypageProfileSilverText.setTextAppearance(requireContext(), tierStyles["Silver"] ?: R.style.TextAppearance_LCK_Light_Detail_10sp)
+                binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles["Silver"] ?: R.style.TextAppearance_Silver)
+            }
+            "Gold" -> {
+                binding.viewMypageProfileCircleGold.setBackgroundResource(tierBackgrounds["Gold"] ?: R.drawable.shape_circle_gold)
+                binding.tvMypageProfileGoldText.setTextAppearance(requireContext(), tierStyles["Gold"] ?: R.style.TextAppearance_LCK_Light_Detail_10sp)
+                binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles["Gold"] ?: R.style.TextAppearance_Gold)
+            }
+            "Master" -> {
+                binding.viewMypageProfileCircleMaster.setBackgroundResource(tierBackgrounds["Master"] ?: R.drawable.shape_circle_master)
+                binding.tvMypageProfileMasterText.setTextAppearance(requireContext(), tierStyles["Master"] ?: R.style.TextAppearance_LCK_Light_Detail_10sp)
+                binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles["Master"] ?: R.style.TextAppearance_Master)
+            }
+            "Challenger" -> {
+                binding.viewMypageProfileCircleChallenger.setBackgroundResource(tierBackgrounds["Challenger"] ?: R.drawable.shape_circle_challenger)
+                binding.tvMypageProfileChallengerText.setTextAppearance(requireContext(), tierStyles["Challenger"] ?: R.style.TextAppearance_LCK_Light_Detail_10sp)
+                binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles["Challenger"] ?: R.style.TextAppearance_Challenger)
+            }
+        }
```

Comment:
```suggestion
binding.viewMypageProfileCircleBronze.setBackgroundResource(tierBackgrounds[tier]!!)
        binding.tvMypageProfileBronzeText.setTextAppearance(requireContext(), tierStyles[tier]!!)
        binding.tvMypageProfileTier.setTextAppearance(requireContext(), tierStyles[tier]!!)
```
map에 직접 넣어줘서 null일 경우가 없으니 차라리 !! 연산자로 null 무시하고 코드 길이 줄이는 게
좋을거 같아여!!